cluster/src/main/java/com/hartwig/pipeline/tertiary/orange/Orange.java

```diff
@@ -255,7 +261,7 @@ private List<BashCommand> buildCommands(final SomaticRunMetadata metadata) {
     @Override
     public VirtualMachineJobDefinition vmDefinition(final BashStartupScript bash, final ResultsDirectory resultsDirectory) {
         return VirtualMachineJobDefinition.builder()
-                .name(NAMESPACE)
+                .name(NAMESPACE_NO_GERMLINE)
```

Comment:
Shouldn't this be `namespace()` rather than the hardcoded NO_GERMLINE?

Reply:
it should!